pom.xml

```diff
@@ -114,6 +114,7 @@
         <bt-args4j-version>2.33</bt-args4j-version>
         <bt-commons-io-version>2.6</bt-commons-io-version>
         <bt-commons-lang-version>2.6</bt-commons-lang-version>
+        <bt-dependency-check-maven-version>6.0.3</bt-dependency-check-maven-version>
```

Comment:
Please rename this property to match the usual pattern for Maven plugin version number properties, and alphabetize it in the “Consolidating maven plugin versions here” section.

Reply:
Do you want `dependency-check-maven-version`?